### PR TITLE
Added some drag and drop example snippets 

### PR DESCRIPTION
**Pull request by Enzo-Demeulenaere**
You can find those examples in the class `BlDragAndDropExamples` under the 'experimentations tag'

There is also the beginning of a new DragHandler prototype only present behind the `Bloc-DragNDrop>> core-experiment` tag but some of the examples use this prototype such as `experimentDragStrategies` for instance.

### CODE REVIEW

File: src/Bloc-Alexandrie/BlBorder.extension.st [hunk 1]
```diff
@@ -20,14 +19,12 @@ BlBorder >> aeApplyTo: aeCanvas element: aBlElement [
...
-			offset: style dashOffset.
+		"No dashes is the default"
+		style dashArray = aBlElement parent border style dashArray ifFalse: [
+			aeCanvas setDashes: style dashArray offset: style dashOffset ].
 
 		aBlElement geometry hasJoins ifTrue: [
-			style lineJoin
-				aeApplyTo: aeCanvas
-				style: style ].
+			style lineJoin aeApplyTo: aeCanvas style: style ].
```
Comment (tinchodias) [inline src/Bloc-Alexandrie/BlBorder.extension.st:27]: This is not related to the drag and drop examples

Comment (Enzo-Demeulenaere) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: No I agree, I just don't know why iceberg decided to change this file although it's only formatting the message on a single line

Comment (tinchodias) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: But not only format, it also adds the `ifFalse:`. Mmm what's the most practical way to remove this diff. I'll check

Comment (Enzo-Demeulenaere) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: ah yes indeed sorry I didn't see it in the embed comment but yes these are changes that were previously in Bloc and my image was older than the time you changed it, I tried to pull the most recent commits and not make these appear in my PR but it appears I still have some things to learn about git and how to deal with this kind of issue

Comment (Enzo-Demeulenaere) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: actually, can I put my changes on another branch of my repo, delete the changes on this BlBorder class and do another PR ? I think we'd lose the history of my commits but it's not **that** important

Comment (Enzo-Demeulenaere) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: Hi Martin, can you help me on this pls ?

Comment (tinchodias) [reply src/Bloc-Alexandrie/BlBorder.extension.st:27]: hi @Enzo-Demeulenaere yes let me check!

**Comment by tinchodias** [comment]
I fixed the merge conflict and did some fixes here on the web UI. It looks ready for merge.

**Comment by Enzo-Demeulenaere** [comment]
Ok thank you Martin, I'll be more careful next time

**Comment by tinchodias** [comment]
No problem. What's the plan now? some new classes have New suffix, so I
guess you are planing to do some refactorings

Martín

**Comment by Enzo-Demeulenaere** [comment]
Yes we're trying to see how can we have a drag and drop implementation that doesn't use feenk's implementation but so far we talked about how difficult it is to abstract a drag and drop handler in a single class as there are lots of behaviors that are up to the developer so it's in discussion 
We might keep the BlPullHandler and add other handlers to provide different drag handlers and widen the possible scenarios
